rero_ils/modules/patron_transaction_events/serializers/csv.py

```diff
@@ -114,6 +115,15 @@ def generate_csv():
         return stream_with_context(generate_csv())
 
 
+def _skip_if_no_record(func):
```

Comment:
A method starting with '_' is considered as private. What does it means for a function?

Reply:
This function can be generalized and added as a generic decorator.

Reply:
@jma due to the default return statement, I think this decorator isn't so generic and can't be generalized.